eras/conway/impl/src/Cardano/Ledger/Conway/Rules/GovCert.hs

```diff
@@ -126,7 +127,9 @@ instance
         pure (2, ConwayCommitteeHasPreviouslyResigned keyH)
       k -> invalidKey k
 
-newtype ConwayGovCertEvent era = GovCertEvent (Event (EraRule "GOVCERT" era))
+data ConwayGovCertEvent c =
+    DRepRegistration !(Credential 'DRepRole c)
+  | DRepUnregistration !(Credential 'DRepRole c)
```

Comment:
@smelc lets also add events for hot key authorization and drep update

Reply:
@Jimbo4350> done :+1: